tensorflow_addons/rnn/cell.py

```diff
@@ -799,3 +799,50 @@ def get_config(self):
         }
         base_config = super().get_config()
         return {**base_config, **config}
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+class ZoneoutLSTMCell(keras.layers.LSTMCell):
+    """LSTM cell with recurrent zoneout.
+
+    https://arxiv.org/abs/1606.01305
+    """
```

Comment:
Please, add the description of the arguments of the class (units, zoneout_h, zoneout_c, seed) in the docstring